src/main/java/com/rtbhouse/kafka/workers/impl/queues/QueuesManager.java

```diff
@@ -22,54 +27,76 @@
 
     private static final Logger logger = LoggerFactory.getLogger(QueuesManager.class);
 
-    private final Long queueTotalSizeBytesToPause;
-    private final long queueSizeBytesToPause;
-    private final long queueSizeBytesToResume;
+    private final WorkersConfig config;
+
+    private final long queueTotalSizeBytesToPause;
```

Comment:
We allow to exceed queueSizeBytesToPause limit for each queue, since the additional global limit seems to be still valid.

Reply:
Ack.
1. A comment would be helpful (with explanation that it is only 'soft' limit and why) 
2. Maybe we can think about configuring another ratio in second case (now or in the future) + making it unbounded sometimes (it would be useful for tuning purposes + it is a bit risky to pause all partitions when its processing depend on each other because it could lead to deadlock)

Reply:
Comment added